openfasoc/generators/ldo-gen/blocks/sky130hvl/manual_macro.tcl

```diff
@@ -3,4 +3,4 @@ cap_2 N 30 80
 cap_3 N 83 30
 cap_4 N 191 30
 cap_5 N 137 30
-vref_gen N 90 80
+vref_gen N 92 80
```

Comment:
Yes, I checked DEF file after macro place.

Reply:
thanks. I think I told @lulu9312 a long time ago. We need to find a fixed placement for the cells we would see on new LDO configurations so that we don't have to deal with it if the number of switches is higher etc..